tiny_dnn/core/params/params.h

```diff
@@ -24,6 +25,7 @@ class Params {
   fully_params fully() const;
   maxpool_params &maxpool();
   global_avepool_params &global_avepool();
+  recurrent_cell_params recurrent_cell() const;
```

Comment:
The return value type should be referenced type.
Please check current HEAD files such as this.
https://github.com/tiny-dnn/tiny-dnn/blob/7d5ce1307a38da18641ee51dba72ecdaba0da4cf/tiny_dnn/core/params/params.h#L19

Maybe you should use `git rebase master` on your PR branch.